package.json

```diff
@@ -18,7 +19,8 @@
     "allowedTokens": "./scripts/allowedTokens",
     "prettier:fix": "npx prettier --write --ignore-unknown .",
     "prettier": "npx prettier --check .",
-    "test": "truffle test --network regtest"
+    "test": "truffle test --network regtest",
+    "test-waffle": "NODE_ENV=test mocha"
```

Comment:
what's the difference between `test` and `test-waffle`. I think we should update the readme to include it

Reply:
keeping test:truffle for contract migration. Maybe we should change this though...